.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  PHPUnit:
+    runs-on: ubuntu-18.04
```

Comment:
Why not `ubuntu` latest?

Reply:
Because `ubuntu-latest` is a moving target that can potentially break our test suite when the it starts referencing a newer version in the future. I've updated this to the correct `ubuntu-20.04` which should give us ample time (5 years LTS + extended support). For the reference, legacy HHVM can only be installed on `ubuntu-18.04` which would match the latest release at the time of writing this.

Reply:
And in fact I've had to downgrade this to `ubuntu-18.04` again because we can't test legacy TLS 1.0 otherwise: https://github.com/clue-labs/reactphp/runs/1757561328?check_suite_focus=true#step:5:50

The longer-term goal would be to fix (skip) the affected tests similar to https://github.com/reactphp/socket/pull/229 and then upgrade to `ubuntu-20.04` again in the future. It's something I'm happy to look into once this PR is in :+1:

Reply:
👍